hw_diag/diagnostics/ecc_diagnostic.py

```diff
@@ -1,6 +1,7 @@
 import json
 from hm_pyhelper.miner_param import get_gateway_mfr_test_result
 from hm_pyhelper.diagnostics.diagnostic import Diagnostic
+from hw_diag.diagnostics.exceptions import EccMalfunctionException
```

Comment:
Should import the original exception: https://github.com/NebraLtd/hm-pyhelper/blob/master/hm_pyhelper/exceptions.py#L9

Reply:
Must've been from before, I changed this to be pulling from hm-pyhelper instead